Library/Homebrew/cli/parser.rb

```diff
@@ -201,32 +222,22 @@ def switch(*names, description: nil, replacement: nil, env: nil, depends_on: nil
       end
       alias switch_option switch
 
-      def value_for_env(env)
-        return if env.blank?
-
-        method_name = :"#{env}?"
-        if Homebrew::EnvConfig.respond_to?(method_name)
-          Homebrew::EnvConfig.public_send(method_name)
-        else
-          ENV.fetch("HOMEBREW_#{env.upcase}", nil)
-        end
-      end
-      private :value_for_env
```

Comment:
Moved this one under the other private methods, again following class layout conventions